test/cypress/integration/dialog-extension.spec.js

```diff
@@ -69,6 +69,12 @@ context('Dialog extension', () => {
     })
   })
 
+  it('verifies sdk.location.is dialog', () => {
+    cy.getSdk(iframeDialogSelector).then(sdk => {
+      expect(sdk.location.is('dialog')).to.equal(true)
```

Comment:
let's test it's `false` for other values

Reply:
Should your suggestion also apply to the other tests below?

Reply:
https://github.com/contentful/ui-extensions-sdk/pull/238#pullrequestreview-278198669 ;)

Reply:
done, thanks!